app/services/bus_predictions_fetcher.rb

```diff
@@ -0,0 +1,15 @@
+class BusPredictionsFetcher
+  require 'net/http'
```

Comment:
Move this line outside the class

Reply:
Will do. And these 4 fetcher services are repetitive so should refactor at some point.